src/features/pages/home/markets/bottom-text/index.tsx

```diff
@@ -0,0 +1,21 @@
+import React from 'react'
+import { bottom_text_wrapper } from './styles.module.scss'
+import Typography from 'features/components/atoms/typography'
+import { Localize } from 'components/localization'
+import Container from 'features/components/atoms/container'
+import Button from 'features/components/atoms/button'
+
+const BottomText = () => {
+    return (
+        <Container.Fluid className={bottom_text_wrapper} pt="10x" md={{ pt: '19x' }}>
```

Comment:
`Container` extends from `Box` and we can use property align from here
```suggestion
        <Container.Fluid align={'center'} pt="10x" md={{ pt: '19x' }}>
```